include/clink/kernels/opdefs/clink_kernels.td

```diff
@@ -87,39 +91,40 @@ def SquareF64Op: Clink_Op<"square.f64"> {
   let verifier = ?;
 }
 
-def OneHotEncoderLoadOp: Clink_Op<"onehotencoder_load"> {
-  let summary = "onehotencoder_load operation";
+class LoadOp<string suffix> : Clink_Op<"load." # suffix> {
```

Comment:
Is it possible to just use the kernel `clink.load` to load arbitrary model without the suffix? Maybe see `tfrt_data.iterator_get_next` for example. Then we can re-use this kernel to load all model subclasses.

Reply:
This PR has already applied this principle to `clink.transform` kernel, but I cannot imagine how we can apply it to `clink.load`. Unlike Java, with which Flink ML can load arbitrary `Stage` according to the classname string, C++ does not have reflections. TFRT also separated the process to create `Dataset` into different kernels like `MakeBatchDataset`, `MakeMemoryDataset`, `MakeFilterDataset`, etc.

Reply:
I see. That makes sense.

Could we name this kernel as `clink.load_onehotencoder` instead of `clink.load.onehotencoder`? Note that we use `{dialect_name}.{kernel_name}.{attribute_value}` when the `attribute_value` is a template parameter value of the corresponding kernel's C++ implementation. For example,  the C++ implementation of the kernel `tfrt_data.batch_dataset.i32` uses `int32_t` as the value of its template parameter `T`.

Reply:
Thanks for the explanation.